cmd/containerd-stargz-grpc/main.go

```diff
@@ -97,9 +102,21 @@ func main() {
 		log.G(ctx).WithError(err).Fatalf("snapshotter is not supported")
 	}
 
-	rs, err := service.NewStargzSnapshotterService(ctx, *rootDir, &config.Config)
-	if err != nil {
-		log.G(ctx).WithError(err).Fatalf("failed to configure snapshotter")
+	var rs snapshots.Snapshotter
+	if *enableFuseManager {
+		fs, err := fusemanager.NewManagerClient(ctx, *rootDir, *fuseManagerAddr, &config.Config)
+		if err != nil {
+			log.G(ctx).WithError(err).Fatalf("failed to configure fuse manager")
+		}
+		rs, err = snbase.NewSnapshotter(ctx, *rootDir, fs, snbase.AsynchronousRemove)
+		if err != nil {
+			log.G(ctx).WithError(err).Fatalf("failed to configure snapshotter")
+		}
+	} else {
+		rs, err = service.NewStargzSnapshotterService(ctx, *rootDir, &config.Config)
+		if err != nil {
+			log.G(ctx).WithError(err).Fatalf("failed to configure snapshotter")
+		}
```

Comment:
Can we print info log telling whether current mode is fusemanager mode or builtin mode?